lib/cdo/chat_client.rb

```diff
@@ -24,23 +24,11 @@ def self.message(room, message, options={})
     post_to_hipchat(room, message, options)
 
     channel = "\##{Slack::CHANNEL_MAP[room] || room}"
-    Slack.message slackify(message.to_s), channel: channel, username: @@name, color: options[:color]
-  end
-
-  def self.slackify(message)
-    # format with slack markdownish formatting instead of html
-    # https://slack.zendesk.com/hc/en-us/articles/202288908-Formatting-your-messages
-    message.strip!
-    message = "```#{message[7..-1]}```" if message =~ /^\/quote /
-    message.
-      gsub(/<\/?i>/, '_').
-      gsub(/<\/?b>/, '*').
-      gsub(/<\/?pre>/, '```').
-      gsub(/<a href=['"]([^'"]+)['"]>/, '<\1|').
-      gsub(/<\/a>/, '>').
-      gsub(/<br\/?>/, "\n")
+    Slack.message message.to_s, channel: channel, username: @@name, color: options[:color]
```

Comment:
TODO(asher): Determine whether the typecast to a string is required. It makes sense that all `message`s should be a string.